### PR TITLE
Fix dockerfile

### PR DESCRIPTION
**Pull request by mscherer**

### CODE REVIEW

**Comment by jberkus** [comment]
@mscherer, @cgwalters: I'm not clear on rpm-config vs buildsys-build.  Please explain?

**Comment by mscherer** [comment]
buildsys-build is a yum group (since it start by a @ ), which contains some rpms (and so install them when the group is installed), among them gcc and redhat-rpm-config.

**Comment by jberkus** [comment]
@mscherer so, I'm really not sure what to do with this PR.

**Comment by mscherer** [comment]
Well, if you ask me, i would say "merge it" :) (but I suppose you wouldn't expect a different answer)

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
-FROM fedora:22
-MAINTAINER mscherer@redhat.com
+FROM fedora:23
+MAINTAINER jberkus@redhat.com
```
Comment (bexelbie) [inline Dockerfile]: Perhaps this should be one of the project atomic generic addresses?

**Comment by bexelbie** [comment]
docker.sh comments need updating.  @mscherer do you need a PR against your PR?

**Comment by bexelbie** [comment]
Also, should we be doing this with a software collection container instead?

**Comment by bexelbie** [comment]
Also, I am still getting the weird git hang with this version:

```
> docker run --rm -p 4567:4567 -v "$(pwd)"/source:/tmp/source:ro -v "$(pwd)"/data:/tmp/data middleman
== The Middleman is loading

Updating git submodules...
fatal: Not a git repository (or any of the parent directories): .git

fatal: Not a git repository (or any of the parent directories): .git
```

File: Dockerfile (hunk 1 above)
Comment (mscherer) [inline Dockerfile]: I am not sure we have generic projectatomic address. The domain is handled by IT, so we would have to see with them to add the alias. But then, that's more question of who have the ownership of the file in the sense "who is tasked to keep it updated".

**Comment by mscherer** [comment]
I do not think we need a software collection, it might make thing more complex (ie, with Fedora, we just have to ping one single group, Fedora, if something is broken ). 
And when i wrote the file, I wanted to have it close as possible to what people are running (in the start, it was built using garrett laptop as reference, then I install the builder on Fedora since they are throw away VM).

File: Dockerfile (hunk 1 above)
Comment (bexelbie) [inline Dockerfile]: I was thinking more of somethign like atomic-devel or similiar.  But it isn't a big deal either way.

**Comment by bexelbie** [comment]
I mentioned software collections mostly because they seemed to be positioned as the preferred way to think about development.  If we don't use one, that is fine too.  I'd just like to see one used ... somewhere :)

**Comment by bexelbie** [comment]
I tested with @buildsys-build and it makes a larger image (+30MB) - are the "size" wars of docker containers relevant here?

@mscherer I sent a PR to your repo with some other ideas around the dockerfile

**Comment by jberkus** [comment]
@bexelbie those git errors are normal, they're a result of some code which looks to update submodules, and should probably be removed.  But middleman is running and can be accessed.

**Comment by bexelbie** [comment]
@jberkus I got a hang, no further movement.

**Comment by jberkus** [comment]
@bexelbie mind opening a seperate issue for that?

**Comment by jberkus** [comment]
Works, merging.

**Pull request merged**